lib/extended-css-selector.js

```diff
@@ -208,7 +210,13 @@ var ExtendedSelector = (function () { // jshint ignore:line
     };
 
     // Constructor
-    function ExtendedSelector(selectorText, tokens) {
+    /**
+     * 
+     * @param {string} selectorText 
+     * @param {Array<string>} tokens 
```

Comment:
Wrong type (and name). It is actually `groups` -- an array of tokens arrays 

1. [ ] Fix the constructor comment